src/lib.rs

```diff
@@ -54,6 +54,15 @@ extern crate jsonrpc_core;
 #[cfg(feature="server")]
 extern crate tempdir;
 
+#[macro_use]
+//#[cfg(features="server")]
```

Comment:
I have no idea why, but it would not compile with `#[cfg(features="server")]`.

I thought it might have to do with stacking the macro and cfg, but I have no idea. Any idea why this would be? I'm compiling with `--features server`